rest_framework/generics.py

```diff
@@ -290,3 +291,20 @@ def patch(self, request, *args, **kwargs):
 
     def delete(self, request, *args, **kwargs):
         return self.destroy(request, *args, **kwargs)
+
+
+def paginated_response(request, queryset, serializer_class, pagination_class=None, **kwargs):
+    if pagination_class is None:
+        pagination_class = api_settings.DEFAULT_PAGINATION_CLASS
+
+    paginator = pagination_class()
+    for attr, value in kwargs.items():
+        setattr(paginator, attr, value)
```

Comment:
Why `setattr` here? In the CBV case we'd expect users to subclass a pagination class if they wanted some differing behavior.